src/components/TSAutocomplete/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react';
+import { AutoComplete } from 'antd';
+import PropTypes from 'prop-types';
+import './index.less';
+
+const dataSource = ['TP.Hồ Chí Minh', 'Hà Nội', 'Hải Phòng'];
```

Comment:
Data source should be passed in via props. Not hardcode inside component => Cannot reuse